Dockerfile

```diff
@@ -35,6 +35,8 @@ RUN cd /app && \
     make && \
     make install
 
+RUN find /app -type f -name '*.pl' -exec sed -i 's/\r$//' {} \;
```

Comment:
can we change the `script/generate.pl` directly just remove the `\r` instead of running this every time?

Reply:
actually not seeing a `\r` here - i'll look into this some more.